blocks/rich-text/index.js

```diff
@@ -387,6 +400,19 @@ export class RichText extends Component {
 		this.context.onCreateUndoLevel();
 	}
 
+	getContainerNode() {
+		// Find the parent "relative" or "absolute" positioned container
+		const findRelativeParent = ( node ) => {
+			const style = window.getComputedStyle( node );
+			if ( style.position === 'relative' || style.position === 'absolute' ) {
+				return node;
+			}
+			return findRelativeParent( node.parentNode );
+		};
+
+		return findRelativeParent( this.editor.getBody() );
```

Comment:
Might it be possible to avoid the wrapped function by e.g. defining `node` with `this.editor.getBody()` if it is not defined? Or perhaps requiring it, don't know what would be better.

Reply:
Now that this function has been removed in https://github.com/WordPress/gutenberg/commit/45221727c4a4283637968792f6a38700697f1a9f we no longer need to worry about this 😃.